bin/middlewares/unsupported_browser.js

```diff
@@ -1,5 +1,20 @@
+function iOSversion(userAgent) {
+  if (/iP(hone|od|ad)/.test(userAgent)) {
+    const v = userAgent.match(/OS (\d+)_(\d+)_?(\d+)?/);
+    return [parseInt(v[1], 10), parseInt(v[2], 10), parseInt(v[3] || 0, 10)];
+  }
+}
+
 function isUnsupported(userAgent = '') {
-  return userAgent.match('Trident'); // old IE detection
+  // iOS < 10 detection
+  if (iOSversion(userAgent)[0] < 10) {
```

Comment:
Instead of putting some low-level implementation detail here (the `[0]`), maybe you could rename the function `isOldSafari` and check this inside.
That way you know you can put `return false;` in your `try/catch` and this function can be rewritten more simplfy, without redundant comments
```js
function isUnsupported(userAgent = '') {
    return isOldSafari(userAgent) || isOldIE(userAgent);
}
```

Reply:
hi, I made the code cleaner, but with no try/catch, tell me if you find it robust enough :)